dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1714,6 +1720,37 @@ def test_datetime_std_across_axis1_null_results(skipna, numeric_only):
         assert_eq(result, expected)
 
 
+@pytest.mark.parametrize(
+    "dtypes",
+    [
+        pytest.param(
+            ("int64[pyarrow]", "float64[pyarrow]"),
+            marks=pytest.mark.skipif(
+                pa is None or not PANDAS_GT_150,
+                reason="requires pyarrow installed and ArrowDtype",
+            ),
+        ),
+        ("Int64", "Float64"),
+    ],
+)
+@pytest.mark.parametrize("func", ["std", "var", "skew"])
+def test_reductions_with_pandas_and_arrow_ea(dtypes, func):
+    if func in ["skew"]:
+        if not scipy:
+            pytest.skip()
+        elif "pyarrow" in dtypes[0]:
+            pytest.xfail("skew not implemented for arrow dtypes")
+
+    pdf = pd.DataFrame({"a": [1, 2, 3, 4], "b": [4, 5, 6, 7]}).astype(
+        {"a": dtypes[0], "b": dtypes[1]}
+    )
+    ddf = dd.from_pandas(pdf, npartitions=1)
+    pd_result = getattr(pdf, func)()
+    dd_result = getattr(ddf, func)()
+
+    assert_eq(dd_result, pd_result, check_dtype=False)  # _meta is wrongly NA
```

Comment:
This seems like a bug -- is there an issue for this? (Can be fixed separately)

Reply:
Not yet, because the bug is hidden by the bug fixed here. Planned on opening one when this is in because creating a reproducer is tricky right now